packages/tools/examples/PolySegWasmContourToSurface/index.ts

```diff
@@ -0,0 +1,310 @@
+const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
+const segmentationId = 'MY_SEGMENTATION_ID';
+
+// ======== Set up page ======== //
+
+const size = '500px';
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.display = 'flex';
+viewportGrid.style.display = 'flex';
+viewportGrid.style.flexDirection = 'row';
+
+const element1 = document.createElement('div');
+const element2 = document.createElement('div');
+element1.style.width = size;
+element1.style.height = size;
+element2.style.width = size;
+element2.style.height = size;
+
+// Disable right click context menu so we can have right click tools
+element1.oncontextmenu = (e) => e.preventDefault();
+element2.oncontextmenu = (e) => e.preventDefault();
+
+viewportGrid.appendChild(element1);
+viewportGrid.appendChild(element2);
+
+content.appendChild(viewportGrid);
+
+createInfoSection(content, { ordered: true })
+  .addInstruction('Draw a contour segmentation on the left viewport')
+  .addInstruction(
+    'Click on the button to convert the contour segmentation to a volume labelmap segmentation'
+  );
+
+// ============================= //
+let toolGroup1, toolGroup2;
+let renderingEngine;
+const toolGroupId1 = 'ToolGroup_Contour';
+const toolGroupId2 = 'ToolGroup_Labelmap';
+const viewportId1 = 'CT_SAGITTAL_CONTOUR';
+const viewportId2 = 'CT_SAGITTAL_LABELMAP';
+
+const segmentIndexes = [1, 2, 3, 4, 5];
+
+addButtonToToolbar({
+  title: 'Convert contour segmentation to closed surface segmentation',
+  onClick: async () => {
+    await segmentation.addSegmentationRepresentations(toolGroupId2, [
+      {
+        segmentationId,
+        type: csToolsEnums.SegmentationRepresentations.Surface,
+        options: {
+          polySeg: true,
+        },
+      },
+    ]);
+  },
+});
+
+addDropdownToToolbar({
+  labelText: 'Segment Index',
+  options: { values: segmentIndexes, defaultValue: segmentIndexes[0] },
+  onSelectedValueChange: (number) => {
+    segmentation.segmentIndex.setActiveSegmentIndex(
+      segmentationId,
+      Number(number) as number
+    );
+  },
+});
+
+addToggleButtonToToolbar({
+  title: 'Show 3D Anatomy',
+  defaultToggle: false,
+  onClick: async (toggle) => {
+    const viewport3 = renderingEngine.getViewport(viewportId2);
+    const volumeActor = viewport3.getDefaultActor().actor as Types.VolumeActor;
+
+    const visibility = toggle;
+    volumeActor.setVisibility(visibility);
+
+    // seems like we should reset camera, most likely this is clipping planes
+    // not critical but we should fix it later
+    viewport3.resetCamera();
+    viewport3.render();
+  },
+});
+
+/**
+ * Runs the demo
+ */
+async function run() {
+  // Init Cornerstone and related libraries
+  await initDemo();
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(PanTool);
```

Comment:
Please use standard tools, and then also add a interpolation tool as default so that it is easy to add a volume here.